CMakePresets.json

```diff
@@ -38,14 +39,26 @@
     },
     {
       "name": "release",
+      "inherits": ["default"],
       "displayName": "Release build",
       "description": "Release build with Ninja generator",
       "generator": "Ninja",
-      "binaryDir": "${sourceDir}/build-release",
       "cacheVariables": {
         "CMAKE_BUILD_TYPE": "Release",
-        "CMAKE_COMPILE_WARNING_AS_ERROR": "OFF",
-        "CMAKE_INSTALL_PREFIX": "/opt/ats",
+        "CMAKE_COMPILE_WARNING_AS_ERROR": "ON",
+        "BUILD_EXPERIMENTAL_PLUGINS": "ON"
+      }
+    },
+    {
+      "name": "reldbg",
+      "inherits": ["default"],
+      "displayName": "Release build with debug info",
+      "description": "Release build with Ninja generator and debug info",
+      "generator": "Ninja",
+      "binaryDir": "${sourceDir}/build-reldbg",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "RelWithDebInfo",
+        "CMAKE_COMPILE_WARNING_AS_ERROR": "ON",
```

Comment:
This flag and `COMPILE_WARNING_AS_ERROR` property were both added in CMake 3.24. They will have no effect on any CI builds running an earlier version. I think this should be documented somewhere, as it is a potential weak link in our CI if we forget about it.